### PR TITLE
fix(savings-plan): discovery error, display text cell

### PR DESCRIPTION
**Pull request by lionel95200x**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/savings-plan-epic` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [ ] Branch is up-to-date with target branch
- [ ] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx [hunk 1]
```diff
@@ -121,6 +122,23 @@ const getInstanceDisplayName = (
   }
 };
 
+const useDefaultOfferId = (pricingByDuration: PricingByDurationType[]) => {
```
Comment (Eric-ciccotti) [inline packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx:125]: maybe you should move this into a dedicated folder

File: packages/manager/apps/pci-savings-plan/src/utils/savingsPlan.test.ts [hunk 2]
```diff
@@ -6,12 +6,18 @@ describe('isValidSavingsPlanName', () => {
     expect(isValidSavingsPlanName('anotherValidName')).toBe(true);
     expect(isValidSavingsPlanName('123456')).toBe(true);
     expect(isValidSavingsPlanName('123_a-456')).toBe(true);
+    expect(
+      isValidSavingsPlanName(
+        'Savings-Plan-GP-2024-09-05-dedsqdqsdqsdqsdsqdqsdqsdsqdqsdsqD',
```
Comment (Eric-ciccotti) [inline packages/manager/apps/pci-savings-plan/src/utils/savingsPlan.test.ts:11]: maybe remove "dedsqdqsdqsdqsdsqdqsdqsdsqdqsdsqD" or change it

File: packages/manager/apps/pci-savings-plan/src/utils/formatter/date.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const formatDate = (date: Date) => date.toISOString().split('T')[0];
```
Comment (fredericvilcot) [inline packages/manager/apps/pci-savings-plan/src/utils/formatter/date.ts:1]: Maybe you could add a little test suite to ensure that makes the job :)

**Review by fredericvilcot — COMMENTED** [review]
Great just a (very) minor remark :)

File: yarn.lock
Comment (Eric-ciccotti) [inline yarn.lock:1]: do we need yarn lock ?

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**